lib/rubocop/cop/style/space_inside_string_interpolation.rb

```diff
@@ -19,25 +19,39 @@ class SpaceInsideStringInterpolation < Cop
         SPACE_MSG = 'Missing space around string interpolation detected.'.freeze
 
         def on_dstr(node)
+          each_style_violation(node) do |final_node, msg|
+            add_offense(final_node, :expression, msg)
+          end
+        end
+
+        private
+
+        def each_style_violation(node)
```

Comment:
Fair enough. I tend to use `each_` to tell that you can expect many of something. If it's only naming, what would you prefer instead?